packages/testing/playwright/tests/ui/24-ndv-paired-item.spec.ts

```diff
@@ -0,0 +1,320 @@
+import { test, expect } from '../../fixtures/base';
+
+test.describe('NDV Paired Items', () => {
+	test.beforeEach(async ({ n8n }) => {
+		await n8n.start.fromBlankCanvas();
```

Comment:
You won't need this part, "fromImportedWorkflow" starts on the canvas of the imported workflow.